Sources/LocalReceiptParsing/DataConverters/UInt8+Extensions.swift

```diff
@@ -43,26 +43,29 @@ extension BitShiftError: CustomStringConvertible {
 
 extension UInt8 {
 
+    /// - Throws: `BitShiftError`
     func bitAtIndex(_ index: UInt8) throws -> UInt8 {
         guard index <= 7 else { throw BitShiftError.invalidIndex(index) }
```

Comment:
should we have this one also throw a specific receiptParser.error?

Reply:
You mean instead of having this sub-error that gets wrapped later?
These are `UInt8` extensions, so they kind of need a specific error, `ReceiptParser.Error` wouldn't make sense.